Comanche055/RESTARTS_ROUTINE.agc

```diff
@@ -109,7 +109,7 @@ TIMETEST	CCS	A		# IS IT AN IMMEDIATE RESTART
 		COUNT	02/RSROU
 
 ITSINDIR	LXCH	GOLOC +1	# GET THE CORRECT E BANK IN CASE THIS IS
-		LXCH	BB		# SWITCHED ERRASIBLE
+		LXCH	BB		# SWITCHED ERASABLE
```

Comment:
❌ `ERRASIBLE` is correct - see [p1415] of the Comanche scans

[p1415]:http://www.ibiblio.org/apollo/ScansForConversion/Comanche055/1415.jpg

Reply:
I was trying to "correct"  those typos, seems I got it wrong.